certification/internal/policy/operator/certified_images.go

```diff
@@ -0,0 +1,98 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/operator-framework/operator-manifest-tools/pkg/image"
+	"github.com/operator-framework/operator-manifest-tools/pkg/pullspec"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/pyxis"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type imageFinder interface {
+	FindImagesByDigest(context.Context, []string) ([]pyxis.CertImage, error)
+}
+
+// CertifiedImagesCheck scans the CSV and validates that all refereenced images are certified.
+type certifiedImagesCheck struct {
+	ImageFinder        imageFinder
```

Comment:
Does `imageFinder` need to be named here?

Reply:
That's a good question. I can try it without it being exported.

Reply:
Apparently not!